src/auth0-session/config.ts

```diff
@@ -287,4 +287,15 @@ export interface LogoutOptions {
    * default in {@link Config.routes.postLogoutRedirect routes.postLogoutRedirect}.
    */
   returnTo?: string;
+
+  /**
+   * Additional custom parameters to pass to the logout endpoint.
+   *
+   * @example pass the federated logout param per https://auth0.com/docs/authenticate/login/logout/log-users-out-of-idps
+   *
+   * ```js
+   * handleLogout(req, res, { logoutParams: { federated: '' } });
```

Comment:
Shouldn't it be `federated: true` instead of an empty string?

Reply:
`federated=true` would work, but Auth0 asks for `https://YOUR_DOMAIN/v2/logout?federated` - [in its docs](https://auth0.com/docs/authenticate/login/logout/log-users-out-of-idps#:~:text=https%3A//YOUR_DOMAIN/v2/logout%3Ffederated) and this is a closer approximation (which also works)

Reply:
```bash
$ node -e "foo=new URL('https://YOUR_DOMAIN/v2/logout'); foo.searchParams.set('federated', ''); console.log(foo.toString())"
https://your_domain/v2/logout?federated=
```